### ISSUE TITLE
Question about Remaining Lessons

### ISSUE THREAD

**Issue by EllenMorningstarUM** [Montana-Media-Arts/441-web-technologies#18]
So official classes end May 4th and finals start May 7th - 11th, which means we have 2 and a half weeks left of the official semester and 6 lessons left in this class? Are we just going to be doing 2 more lessons and calling it good?

**Comment by Br0210** [comment]
THERE WILL PROBABLY BE A GROUP PROJECT    On Tuesday, April 24, 2018, 9:26:31 AM MDT, Ellen Morningstar (Kearns) <notifications@github.com> wrote:  
 
 
So official classes end May 4th and finals start May 7th - 11th, which means we have 2 and a half weeks left of the official semester and 6 lessons left in this class? Are we just going to be doing 2 more lessons and calling it good?

**Comment by michaelmusick** [comment]
No, we will do one more lesson. I will post later this week. Along with any other information.



Michael Musick
--
michaelmusick.com

Media Artist & Electroacoustic Musician
Assistant Professor
School of Media Arts
University of Montana
http://www.umt.edu/mediaarts/
